lib/helpers.coffee

```diff
@@ -158,6 +158,34 @@ module.exports = Helpers =
         )
     return toReturn
 
+  findFileAsync: (directory, names) ->
+    throw new Error "Specify a filename to find" unless arguments.length
+    unless names instanceof Array
+      names = [names]
+
+    chunks = directory.split(path.sep)
+    promise = Promise.resolve(null)
+
+    while chunks.length
+      currentDir = chunks.join(path.sep)
+      break unless currentDir
+
+      do (currentDir) ->
```

Comment:
@iam4x Babel has evolved a lot lately and It's fixed the things I disliked about it, especially with Babel 6.0 the modular transforms are a blast :bomb: